test/unit/test_gitlab_bug.py

```diff
@@ -177,19 +177,30 @@ def test_iterate_merge_requests_wip_setting_absent(mock_prepare_message, _mock_m
 
 
 def test_prepare_message(_mock_merge_request, _mock_url, _mock_user, _mock_repo):
-    result = GitlabBug.prepare_message(_mock_merge_request)
+    result = GitlabBug.prepare_message(_mock_merge_request, _mock_repo)
+    assert f'{_mock_repo["name"][0]}' in result
     assert 'Merge Request' in result
     assert f'{_mock_merge_request["assignees"][0]["web_url"]}|{_mock_merge_request["assignees"][0]["username"]}' in result  # noqa
     assert f'{_mock_merge_request["web_url"]}|{_mock_merge_request["title"]}' in result
 
 
-def test_prepare_message_no_assignees_data(_mock_merge_request):
+def test_prepare_message_no_assignees_data(_mock_merge_request, _mock_repo):
     _mock_merge_request['assignees'][0]['username'] = None
-    result = GitlabBug.prepare_message(_mock_merge_request)
+    result = GitlabBug.prepare_message(_mock_merge_request, _mock_repo)
     assert '*Waiting on:* No assignee' in result
 
 
-def test_prepare_message_no_assignee(_mock_merge_request):
+def test_prepare_message_no_assignee(_mock_merge_request, _mock_repo):
     _mock_merge_request['assignees'] = []
-    result = GitlabBug.prepare_message(_mock_merge_request)
+    result = GitlabBug.prepare_message(_mock_merge_request, _mock_repo)
     assert '*Waiting on:* No assignee' in result
+
+
+@mock.patch('pullbug.gitlab_bug.LOGGER')
+@mock.patch('requests.get', side_effect=requests.exceptions.RequestException('mock-error'))
+def test_get_projects_exception(mock_request, mock_logger):
```

Comment:
Nice! We'd probably also want a test that ensures this was successful in addition to the exception.